internal/event/handler.go

```diff
@@ -1,11 +1,29 @@
 package event
 
-import "github.com/metal-stack/metal-core/pkg/domain"
+import (
+	"time"
+
+	"github.com/metal-stack/metal-core/pkg/domain"
+)
 
 type eventHandler struct {
 	*domain.AppContext
+	sr chan switchReconfigureEvent
+}
+type switchReconfigureEvent struct {
+	switchName string
+	eventType  string
+	occurence  time.Time
+}
+
+func newSwitchReconfigureEvent(switchName, eventType string) switchReconfigureEvent {
+	return switchReconfigureEvent{
+		switchName: switchName,
+		eventType:  eventType,
+		occurence:  time.Now(),
+	}
 }
 
 func NewHandler(ctx *domain.AppContext) domain.EventHandler {
-	return &eventHandler{ctx}
+	return &eventHandler{ctx, make(chan switchReconfigureEvent, 50)}
```

Comment:
Why 50 ? At least factor out to a const with meaningful name

Reply:
Okay, then we'll use a timer based reconfiguration only.

Reload happens only when the checksum of frr.conf or interfaces changed: see https://github.com/metal-stack/metal-networker/blob/master/pkg/net/applier.go#L61